src/com/xilinx/rapidwright/util/FileTools.java

```diff
@@ -409,38 +407,30 @@ public static String readString(DataInputStream dis){
 	@SuppressWarnings("resource")
 	public static Object loadFromFile(String fileName){
 		File inputFile = new File(fileName);
-		FileInputStream fis;
-		BufferedInputStream bis;
-		ObjectInputStream ois;
-		Object o; 
-		try {
-			fis = new FileInputStream(inputFile);
-			bis = new BufferedInputStream(fis);
-			ois = new ObjectInputStream(bis);
-			o = ois.readObject();
-			ois.close();
-			bis.close();
-			fis.close();
-		} 
+
+		try (FileInputStream fis = new FileInputStream(inputFile);
+			 BufferedInputStream bis = new BufferedInputStream(fis);
+			 ObjectInputStream ois  = new ObjectInputStream(bis)){
```

Comment:
```suggestion
			BufferedInputStream bis = new BufferedInputStream(fis);
			ObjectInputStream ois  = new ObjectInputStream(bis)){
```
Remove stray spaces.